graphics/partio/Portfile

```diff
@@ -30,3 +32,24 @@ depends_build-append \
                     port:doxygen \
                     port:swig-python
 depends_lib         port:zlib
+
+post-destroot {
+    if {![file exists ${destroot}${prefix}/share/${name}/py]} {
+        xinstall -d ${destroot}${prefix}/share/${name}/py
+    }
+    copy ${worksrcpath}/src/py/example \
+         ${destroot}${prefix}/share/${name}/py
+}
+
+variant tests description {Build tests} {
+    depends_lib-append      port:gtest
+    configure.args-append   GTEST_ENABLED=ON
+    test.run    yes
+
+    post-destroot {
+        copy ${worksrcpath}/src/data \
+             ${destroot}${prefix}/share/${name}/test
+    }
+}
```

Comment:
When the unit tests are built, they get included in the final products, which means that users are able to run the unit tests manually. Adding the data files means that if a user wants to run a unit test manually, they can use the data files as input, instead of having to come up with their own input file.

Reply:
I think that's fair enough, thank you for explaining. Do you think it's worth adding a `notes-append` about this in the test variant, explaining how to do this?

Reply:
I've been reprimanded in the past for overusing notes, for info which aren't considered critical for the user who is installing the package to know about.

The [Partio documentation](http://partio.us/documentation.html) doesn't refer to the tests directly, but does sort of indirectly reference the tests by essentially using the unit tests in their usage examples and the python and C++ tutorials sections. Frankly, it would probably be a better idea to get the upstream authors to beef up their documentation, than to have to provide a `notes-append` ourselves.